conf/vagrant/provisioning/roles/https/templates/san.cnf

```diff
@@ -0,0 +1,17 @@
+[req]
+distinguished_name = req_distinguished_name
+req_extensions     = v3_req
+x509_extensions    = v3_req
+
+[req_distinguished_name]
+commonName       = {{ hostname }}
+organizationName = Palantir.net, Inc.
+localityName     = Evanston
+countryName      = US
+
+[v3_req]
+# The extentions to add to a self-signed cert
+subjectKeyIdentifier = hash
+basicConstraints     = critical,CA:false
+subjectAltName       = DNS:{{ hostname }},DNS:www.{{ hostname }}
```

Comment:
Should this line include `*.{{ hostname }}`?

Also, should we pass the `extra_hostnames` array through to Ansible in the Vagrantfile, and include those here?

Reply:
I'm not sure if * would work, but that's definitely possible. I forgot about `extra_hostnames`... we might need to do some processing to prefix them all with `DNS:` but if you can do that easily that'd be awesome.